### ISSUE TITLE
Error reported for temp data.npy file

### ISSUE THREAD

**Issue by aidanacquah** [OxWearables/actipy#41]
When running actipy.read_device() an error is reported:

"Error: path/to/data.npy - The process cannot access the file because it is being used by another process."

It should be noted that this is a non breaking error and actipy continues to work as expected when ignoring this error.

**Comment by chanshing** [comment]
Can you share the details of the machine you used to reproduce this error? First time seeing it

**Comment by aidanacquah** [comment]
Produced when running actipy on Windows 11 machine. 
Reproduced on several different machines.

Conda environment set up with:
Python = 3.9
openjdk = 11.0.13
pip = 24.0

actipy=3.0.8

Using ipython, I run the commands:
```
import actipy
data, info = actipy.read_device("C:/path/to/file.cwa")
```

This produces the error message:
```
Decompressing... Done! (1.19s)
Reading file... Done! (16.32s)
Converting to dataframe... Done! (4.87s)
Error: C:\path\to\AppData\Local\Temp\tmpndeeot4f\data.npy - The process cannot access the file because it is being used by another process.
Lowpass filter... Done! (13.14s)
Gravity calibration... Done! (18.21s)
Nonwear detection... Done! (6.73s)
Resampling... Done! (19.21s)
```

**Comment by claaslendt** [comment]
Even though this issue is already a few months old, I just want to add that I experience the same error everytime I use `actipy.read_device()`. This happens for .cwa files from AX3 and AX6 devices. As Aidan said, this is a non-breaking error and the processing works as expected.

python = 3.10.15 and 3.11.1
actipy = 3.4.0 and 3.0.5

**Comment by chanshing** [comment]
@claaslendt @aidanacquah Thanks for the further details.
I just had access to a win11 setup and could debug this.
I pushed a patch and it should be fixed now in 3.4.1
https://github.com/OxWearables/actipy/releases/tag/v3.4.1

To update: `pip install -U actipy`

**Comment by claaslendt** [comment]
@chanshing That was quick! Just to confirm: the error is gone after updating to the patched version. Thanks!
